docs/xks/developer-guide/security.md

```diff
@@ -9,3 +9,54 @@ Certain policies will always be enforced in the cluster as a guard rail to minim
 any request to create or update a Kubernetes resource will first have to pass through the policy check. If the resource in question does not comform to the policy it will be rejected
 by the API server when applying the change. Knowing this is important as certain features or options documented on the Internet may not be available or are restricted in the XKS service.
 This can include things like certain types of volume mounts, labeling requirements or container capabilities for a Pod.
+
+## Security Features
+
+### OPA Gatekeeper
+
+OPA (Open Policy Agent) Gatekeeper is a product in which we have a set of rules defining how we allow containers to be created. To simplify this for the developers these rules are then applied automatically to all Pods created to achive our set standard.
+You can read more in their documentation: <https://open-policy-agent.github.io/gatekeeper/website/docs/howto/>
+And in our library: <https://github.com/XenitAB/gatekeeper-library>
+
+### SecurityContext
+
+We have applied basic SecurityContext automatically with OPA Gatekeeper to all Pods in our XKS clusters. The basic YAML code is described below. This configuration is applied to both initcontainers and containers and we encourage you to set this up in your own gitops repository to make it more visible for developers.
```

Comment:
It is not "our" cluster, but rather the customer's cluster.

Reply:
> we encourage you to set this up in your own gitops repository to make it more visible for developers.

The reader is a developer. I would suggest "... but you may still want to add this configuration to your manifests to be explicit about the limitations under which containers operate." or some such.

Reply:
I think this text could benefit from being clear about whether these constraints are enforced or are defaults. This is particularly true for `readOnlyRootFilesystem` which may cause problems for e.g. Python developers if they are not aware of it; Python tries to cache byte code files in the same directory as teh corresponding source code file.